css/style.css

```diff
@@ -6,6 +6,118 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
+.logo {
+    width: 45px;
+    height: 45px;
+    float: left;
+    padding-left: 130px;
+}
+
+ul {
+    list-style: none;
+}
+
+
+li {
+    display: inline-block;
+    padding: 10px;
+    margin-left: 25px;
+}
+
+header {
+    justify-content: space-between;
+    text-align: center;
+    font-size: 16px;
+    color: darkslategray;
+
+}
+
+.first-banner {
+    max-width: 100%;
+    padding-bottom: 15px;
+}
+
+.banner {
+    position: relative;
+    text-align: center;
+    color: white;
+}
+
+.introduction {
+    position: absolute;
+    top: 40%;
+    left: 36%;
+    font-weight: 100;
+    font-size: 48px;
+}
+
+.intro-subheading {
+    position: absolute;
+    top: 55%;
+    left: 35%;
+    font-weight: 100;
+}
+
+.learn {
+    background-color: rgb(211, 90, 9);
+    border: none;
+    padding: 10px 10px;
+    position: absolute;
+    top: 65%;
+    left: 45%;
+    font-size: 20px;
+}
+
+.learn:hover {
+    background-color: orangered;
+}
+
+.information {
+    display: flex;
+    justify-content: space-around;
+
+}
+
+
+.details {
+    padding: 10px;
+    margin: auto;
+    text-align: center;
+    font-size: 14px;
+    justify-content: stretch;
+}
+
+.fuel {
+    width: 150px;
+    height: 150px;
+}
+
+.karma {
+    text-align: center;
+    font-size: 48px;
+}
+
+.join {
+    text-align: center;
+}
+
+
+.media {
+    text-align: center;
+}
```

Comment:
These 2 classes do the same thing. You should create one class only, and re-use it several times.